testing/trino-tests/src/test/java/io/trino/execution/TestEventListenerBasic.java

```diff
@@ -468,7 +486,7 @@ public void testReferencedTablesWithColumnMask()
 
         column = table.getColumns().get(1);
         assertThat(column.getColumn()).isEqualTo("test_bigint");
-        assertThat(column.getMasks()).isEmpty();
+        assertThat(column.getMasks()).hasSize(0);
```

Comment:
Why did it change? Is it just a refactor? If so, could you please extract it to another commit? Also, there are multiple occurrences of `assertThat(...).isEmpty();` in this class.

Reply:
It messed up during rebase